notebook.ipynb

```diff
@@ -1,5 +1,12 @@
 {
```

Comment:
<p>Next, it would be good to include a short description and explaination of this last output</p><p> </p><p>Then add a concise conclusion, summarising what the notebook has shown, including the tools, analysis and datasets that have been used</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/CI2023-RC-team2/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>Done.</p>